boxoffice/views/item_collection.py

```diff
@@ -71,3 +71,15 @@ def item_collection(item_collection):
         if category_json:
             categories_json.append(category_json)
     return jsonify(html=render_template('boxoffice.html.jinja2'), categories=categories_json, refund_policy=item_collection.organization.details.get('refund_policy', ''))
+
+
+@app.route('/<org_name>/<item_collection_name>', methods=['GET', 'OPTIONS'])
+# @cors
+@load_models(
+    (Organization, {'name': 'org_name'}, 'organization'),
+    (ItemCollection, {'name': 'item_collection_name'}, 'item_collection')
```

Comment:
ItemCollection must be loaded against the organisation.

Reply:
Fixed.